Makefile

```diff
@@ -38,6 +39,7 @@ clean:
 	@-rm -rf public/components
 	@-rm -rf public/css
 	@-rm -rf public/urth_components
+	@-docker rmi -f $(KG_IMAGE_NAME) $(DASHBOARD_IMAGE_NAME)
```

Comment:
Probably don't want to clean up the docker image every time we clean. The caching should be smart enough to rebuild what's needed. Is that not working for you?